### ISSUE TITLE
Add 'ConcussionGrenade.Throw' and 'ConcussionGrenade.ThrowCancel' to our game sounds

### ISSUE THREAD

**Issue by EthanV-3D** [momentum-mod/game#2087]
### What feature is your improvement idea related to?

Add support for 'ConcussionGrenade.Throw' (left-click throwing sound) and 'ConcussionGrenade.ThrowCancel' (right-click throwing cancel sound) to our sound-script game_sounds_momentum_weapons.txt

Also link up the sounds to the specific actions on the conc weapon when you throw a conc, and when you cancel

### Describe the solution you'd like

something like this pls, tyty
![image](https://github.com/momentum-mod/game/assets/26844021/0f8ab7b3-e8a1-4768-89c9-76fd17a6664d)

### Additional context

Am waiting on this implementation to bring in new game sounds for the conc

**Comment by Gocnak** [comment]
Added in but the sounds are empty currently. Maybe @AiTrat can make some pretty ones? That's a separate issue/todo item
